tests/improved-sanity-test/main.yml

```diff
@@ -38,148 +38,180 @@
 
   roles:
     # This playbook requires Ansible 2.1 and an Atomic Host
-    - { role: ansible_version_check, avc_major: "2", avc_minor: "1", tags: ['ansible_version_check'] }
+    - { role: ansible_version_check, avc_major: "2", avc_minor: "1", tags: ['ansible_version_check','cloud_image'] }
```

Comment:
When reviewing the code, I realized it was a little more difficult to read than it should be.  If its not too much trouble, maybe we can reformat everything in this style to match the consistency of the rest of the code, i.e.

- role: ansible_version_check
  avc_major: "2"
  avc_minor: "1"
  tags:
    - ansible_version_check
    - cloud_image`